hi-ml-cpath/src/health_cpath/utils/deepmil_utils.py

```diff
@@ -89,6 +93,9 @@ def get_encoder(self, outputs_folder: Optional[Path]) -> TileEncoder:
         elif self.encoder_type == Resnet50_NoPreproc.__name__:
             encoder = Resnet50_NoPreproc(tile_size=self.tile_size, n_channels=self.n_channels)
 
+        elif self.encoder_type == SwinTransformer_NoPreproc.__name__:
+            encoder = SwinTransformer_NoPreproc(tile_size=self.tile_size, n_channels=self.n_channels)
```

Comment:
can we cover this encoder in a test?

Reply:
We don't currently have tests that cover any of the encoders, we can maybe address it in a separate PR, but anyways I'm not sure we need to test these as it's external code, we're just import predefined encoders from libraries that already test the encoders expected behaviour

Reply:
ok, thanks, codecov complains so I was wondering if we should add some tests